packages/webapp/public/locales/ml/translation.json

```diff
@@ -533,209 +533,209 @@
   },
   "ENTITY_TYPES": {
     "LOCATION": "സ്ഥലം",
-    "TASK": ""
+    "TASK": "ചുമതല"
   },
   "ERROR_FALLBACK": {
-    "CONTACT": "",
-    "MAIN": "",
-    "RELOAD": "",
-    "SUBTITLE": "",
-    "TITLE": ""
+    "CONTACT": "ഇപ്പോഴും ചെളിയിൽ കുടുങ്ങിയിട്ടുണ്ടോ? വിഷമിക്കേണ്ട, നിങ്ങളെ പുറത്തെടുക്കാൻ ഞങ്ങൾ ഇവിടെയുണ്ട്:\n<1>{{supportEmail}}</1>",
```

Comment:
Here is another newline not present in english

Reply:
Thanks, fixed.